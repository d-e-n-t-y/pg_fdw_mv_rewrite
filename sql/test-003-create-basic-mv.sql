create materialized view test_mv1 as select key, count(value) FROM test GROUP BY key;

refresh materialized view test_mv1;

insert into pgx_rewritable_matviews values ('public', 'test_mv1', array['public.test']);
