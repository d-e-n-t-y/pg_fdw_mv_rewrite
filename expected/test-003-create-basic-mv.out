create materialized view test_mv1 as select key, count(value) FROM test GROUP BY key;
refresh materialized view test_mv1;
select mv_rewrite.enable_rewrite ('test_mv1');
 enable_rewrite 
----------------
 
(1 row)

